hack/tools/tilt-prepare/main.go

```diff
@@ -936,6 +936,10 @@ func getProviderObj(version *string) func(prefix string, objs []unstructured.Uns
 			providerType = string(clusterctlv1.RuntimeExtensionProviderType)
 			providerName = manifestLabel[len("runtime-extension-"):]
 		}
+		if strings.HasPrefix(manifestLabel, "add-on-") {
```

Comment:
this seems to not be matching fmt.Sprintf("addon-%s", name) in cmd/clusterctl/api/v1alpha3/labels.go